feature-libs/customer-ticketing/components/customer-ticketing-details/customer-ticketing-details.component.html

```diff
@@ -0,0 +1,44 @@
+<ng-container *ngIf="ticketDetails$ | async as ticket">
+  <div class="cx-ticket-details">
+    <div class="container">
+      <div class="cx-details-card">
+        <cx-card
+          [content]="
+            prepareCardContent(ticket?.id, 'customerTicketing.ticketId') | async
```

Comment:
in "ticket?.id"  is the compiler asking you to check for null?  It shouldn't be since the container checks for ticketDetails$

Reply:
yes, I believe we can remove it.